langchain4j-mongodb-atlas/src/main/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStore.java

```diff
@@ -268,40 +276,66 @@ public List<String> addAll(List<Embedding> embeddings, List<TextSegment> embedde
     }
 
     @Override
-    public List<EmbeddingMatch<TextSegment>> findRelevant(Embedding referenceEmbedding, int maxResults, double minScore) {
-        List<Double> queryVector = referenceEmbedding.vectorAsList().stream()
+    public void removeAll() {
+        collection.deleteMany(new Document());
```

Comment:
[nit] Just to be more explicit. 
```suggestion
        collection.deleteMany(Filters.empty());
```